cmd/stack/destroy.go

```diff
@@ -35,7 +35,7 @@ func Destroy(ctx context.Context) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "destroy <name>",
 		Short: "Destroy a compose",
-		Args:  utils.MaximumNArgsAccepted(1, "https://www.okteto.com/docs/0.10/reference/cli/#destroy-2"),
+		Args:  utils.MaximumNArgsAccepted(1, "https://www.okteto.com/docs/reference/okteto-cli/"),
```

Comment:
There is not docs page for the push command, so we send them to the okteto cli page